tensorflow_encrypted/layers/batchnorm.py

```diff
@@ -4,15 +4,16 @@
 
 
 class Batchnorm(core.Layer):
-    def __init__(self, mean, variance, scale, offset, variance_epsilone=1e-8):
+    def __init__(self, mean: np.array, variance: np.array, scale: np.array,
+                 offset: np.array, variance_epsilone: float = 1e-8) -> None:
```

Comment:
Is `variance_epsilone` a typo?

Reply:
oups yes, good catch! It should be `variance_epsilon`  https://www.tensorflow.org/api_docs/python/tf/nn/batch_normalization